src/test/workerRunner.ts

```diff
@@ -72,6 +72,8 @@ export class WorkerRunner extends EventEmitter {
     if (this._isStopped)
       return;
     if (this._currentTest) {
+      if ('error' in this._currentTest.testInfo)
```

Comment:
why not `if (this._currentTest.testInfo.error)` ?

Reply:
this pattern was before used a few times before in the file so I followed it. But was also thinking your proposed way should be better.